rnlp/parse.py

```diff
@@ -149,7 +152,18 @@ def makeIdentifiers(blocks, target="sentenceContainsTarget(+SID,+WID).",
                     makeIdentifiers(blocks)
                     # 100%|██████████████████████| 2/2 [00:00<00:00, 18.49it/s]
     """
-
+    # Specify a global variable
+    global OUTPUT_DIR
```

Comment:
We should avoid declaring a `global OUTPUT_DIR` here.

Private modules (e.g., `_writeFact()`) can be executed independently from `makeIdentifiers`, which would lead to an unstable state where `OUTPUT_DIR` is undefined.

I think it would be reasonable to pass `output_dir=` as a keyword argument in `_writeBlock`, `_writeSentenceInBlock`, etc.